gsoc/2020/posts/jose_miguel_sanchez/post01.md

```diff
@@ -1,15 +1,39 @@
-# GSoC'20 RoboComp project: NAME OF THE PROJECT
+# GSoC'20 RoboComp project: Syntax-error highlighter for LearnBlock
  
 DATE
  
 ## About me
  
-SOME INFORMATION ABOUT YOU
-
+Hello! I'm José Miguel, a Computer Engineering student in Cáceres, Spain.
+I've grown up with a passion for technology, and over time I've played around
+with computer graphics, programming language and operating system design,
+game modding, web design and programming, a bit of computer security...
+I love learning something from every field I discover!
 
 ## About the Project
 
-DESCRIPTION OF THE PROJECT AND ITS GOALS
+LearnBlock is a visual robot programming tool which eases the path towards
+learning full-fledged robot programming (in Python) by putting some simpler
```

Comment:
learning full-fledged robot programming (in Python). Putting some more straightforward